externs/browser/w3c_dom1.js

```diff
@@ -422,6 +422,34 @@ Document.prototype.createTextNode = function(data) {};
  */
 Document.prototype.getElementsByTagName = function(tagname) {};
 
+/**
+ * @param {string=} opt_unused1
+ * @param {string=} opt_unused2
+ * @return {undefined}
+ * @see https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#dom-document-open
+ */
+Document.prototype.open = function(opt_unused1, opt_unused2) {};
```

Comment:
You are right - I am not sure how I missed that except I think I looked at the wrong spec 🤷‍♂part.

For reference the gecko WebIDL for the `open` symbol is

```
[CEReactions, Throws]
Document open(optional DOMString type, optional DOMString replace = ""); // type is ignored
[CEReactions, Throws]
WindowProxy? open(DOMString url, DOMString name, DOMString features, optional boolean replace = false);
```

While I have not checked I am assuming that the `google/jsinterop-generator` tool will not gracefully handle this scenario. I will need to run some experiments to try and figure out a strategy. It may be that the symbols other than `open` can be be moved in this PR and `open` will need to wait till the `google/jsinterop-generator` tool is modified. 

Anyhoo I will experiment and ping I have a better idea

Reply:
Ok reverted the type changes to open. I was unable to figure out a good way of typing it that worked with all the tools. 

This should be ready for re-review @tjgq